backup-stores/gcs/src/test/java/io/camunda/zeebe/backup/gcs/util/GcsContainer.java

```diff
@@ -7,20 +7,38 @@
  */
 package io.camunda.zeebe.backup.gcs.util;
 
+import com.github.dockerjava.api.command.InspectContainerResponse;
 import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.images.builder.Transferable;
 import org.testcontainers.utility.DockerImageName;
 
 public final class GcsContainer extends GenericContainer<GcsContainer> {
   private static final DockerImageName IMAGE = DockerImageName.parse("fsouza/fake-gcs-server");
   private static final int PORT = 4443;
+  private static final String STARTER_SCRIPT = "/testcontainers_start.sh";
 
   public GcsContainer() {
     this("1");
   }
 
   public GcsContainer(String version) {
     super(IMAGE.withTag(version));
-    this.withExposedPorts(PORT).withCommand("-scheme http");
+    this.withExposedPorts(PORT)
+        .withCreateContainerCmdModifier(cmd -> cmd.withEntrypoint("sh"))
+        .withCommand(
+            "-c", "while [ ! -f " + STARTER_SCRIPT + " ]; do sleep 0.1; done; " + STARTER_SCRIPT);
+  }
```

Comment:
Lets add a comment why we have this, why it is needed, actual result of our discussion